### ISSUE TITLE
Consider relations in walkthrough

### ISSUE THREAD

**Issue by manfredbrandl** [openstreetmap/iD#4012]
Relations should be considered for training in the walkthrough. Praticing must include Multipolygons (~52% of existing relations) and may include restriction (~13% of existing relations). Relations types route and boundary (each ~9% of existing relations) should be mentioned.

**Comment by bhousel** [comment]
Yes I think I'm going to make an "advanced" walkthrough which teaches about relations.

The target audience for the basic walkthrough is someone at a mapathon, or someone who just heard about OpenStreetMap some other way, but doesn't necessarily care much about the project.

The ideal audience for an "advanced" walkthrough would be someone who has already made a dozen or so edits and wants to be a more productive.

**Comment by manfredbrandl** [comment]
Maybe what we have in the help now is enough documentation.

**Comment by quincylvania** [comment]
@manfredbrandl The written documentation is important, but I'd like to try #6079 and have individual tutorials for all kinds of things like this. Some people have issues with reading or processing text, so an option to "learn by doing" makes iD more accessible to them.

**Comment by manfredbrandl** [comment]
@quincylvania I support #6709 to enhance the written documentation, which is easiert to produce but not so helpfull als a bunch of short walkthrough chapter e.g. on relations.
